mne/viz/evoked.py

```diff
@@ -1529,162 +1689,204 @@ def plot_compare_evokeds(evokeds, picks=list(), gfp=False, colors=None,
     invert_y : bool
         If True, negative values are plotted up (as is sometimes done
         for ERPs out of tradition). Defaults to False.
-    show_sensors: bool | int | None
+    show_sensors: bool | int | str | None
         If not False, channel locations are plotted on a small head circle.
-        If an int, the position of the axes (forwarded to
+        If int or str, the position of the axes (forwarded to
         ``mpl_toolkits.axes_grid1.inset_locator.inset_axes``).
         If None, defaults to True if ``gfp`` is False, else to False.
-    show_legend : bool | int
-        If not False, show a legend. If int, the position of the axes
-        (forwarded to ``mpl_toolkits.axes_grid1.inset_locator.inset_axes``).
+    show_legend : bool | str | int
+        If not False, show a legend. If int or str, it is the position of the
+        legend axes (forwarded to
+        ``mpl_toolkits.axes_grid1.inset_locator.inset_axes``).
+    split_legend : bool
+        If True, the legend shows color and linestyle separately; `colors` must
+        not be None. Defaults to True if ``cmap`` is not None, else defaults to
+        False.
     axes : None | `matplotlib.axes.Axes` instance | list of `axes`
         What axes to plot to. If None, a new axes is created.
         When plotting multiple channel types, can also be a list of axes, one
         per channel type.
     title : None | str
-        If str, will be plotted as figure title. If None, the channel
-        names will be shown.
+        If str, will be plotted as figure title. If None, the channel names
+        will be shown.
     show : bool
         If True, show the figure.
 
     Returns
     -------
     fig : Figure | list of Figures
-        The figure(s) in which the plot is drawn.
+        The figure(s) in which the plot is drawn. When plotting multiple
+        channel types, a list of figures, one for each channel type is
+        returned.
+
+    Notes
+    -----
+    When multiple channels are passed, this function combines them all, to
+    get one time course for each condition. If gfp is True it combines
+    channels using global field power (GFP) computation, else it is taking
+    a plain mean.
+
+    This function is useful for comparing multiple ER[P/F]s - e.g., for
+    multiple conditions - at a specific location.
+
+    It can plot:
+
+        - a simple :class:`mne.Evoked` object,
+        - a list or dict of :class:`mne.Evoked` objects (e.g., for multiple
+          conditions),
+        - a list or dict of lists of :class:`mne.Evoked` (e.g., for multiple
+          subjects in multiple conditions).
```

Comment:
no need to indent, doing so turns it from a standard bullet list to a quoted bullet list